packaging/prover/miden-prover.service

```diff
@@ -7,7 +7,8 @@ WantedBy=multi-user.target
 
 [Service]
 Type=exec
-ExecStart=/usr/bin/miden-proving-service start-worker --host 0.0.0.0 --port 50052 --tx-prover
+EnvironmentFile=/etc/opt/miden-prover/service.conf
+ExecStart=/usr/bin/miden-proving-service start-worker --host 0.0.0.0 --port ${PORT} --${TYPE}-prover
```

Comment:
I the `--host 0.0.0.0` parameter here needed? According to the latest [docs](https://github.com/0xMiden/miden-base/tree/next/bin/proving-service#worker) there is not such parameter. Or are the docs incorrect?

Reply:
Removed all flags now b/c of the env vars. Confirmed the binaries run without flags specified.